src/sage/rings/integer.pyx

```diff
@@ -7163,6 +7163,45 @@ cdef class Integer(sage.structure.element.EuclideanDomainElement):
         else:
             raise ValueError("algorithm must be one of: 'pari' or 'gmp' (alias: 'mpir')")
 
+    def to_bytes(self, length=1, byteorder="big", is_signed=False):
+        """
+        Return an array of bytes representing an integer.
+
+        Internally relies on the python ``int.to_bytes()`` method.
+
+        INPUT:
+
+        - ``length`` -- positive integer (default: ``1``); integer is represented in
+          ``length`` bytes. ``OverflowError`` is raised if the integer is not representable
+          with the given length.
+
+        - ``byteorder`` -- str (default: ``"big"``); determines the byte order of
+          the output; can only be ``"big"`` or ``"little"``
+
+        - ``is_signed`` -- boolean (default: ``False``); determines whether to use two's
+          compliment to represent the integer
+
+        OUTPUT:
+
+        - Bytes representing ``self``
+
+        .. TODO:: It should be possible to convert straight from the gmp type in cython.
+        This could be significantly faster, but I am unsure of the fastest and cleanest
+        way to do this.
```

Comment:
```suggestion
        .. TODO::

            It should be possible to convert straight from the gmp type in cython.
            This could be significantly faster, but I am unsure of the fastest and cleanest
            way to do this.
```
I think this is causing the docbuild issue.